src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -288,6 +296,41 @@ private function registerDoctrineExtensionConfiguration(ContainerBuilder $contai
         }
     }
 
+    private function registerHttpCache(ContainerBuilder $container, array $config, XmlFileLoader $loader)
+    {
+        $loader->load('http_cache.xml');
+
+        if (true !== $config['http_cache']['enable_tags']) {
```

Comment:
It can be intended (I often do that). It allows to always serve fresh data for the first request but reduce the server load/bandwidth using expiration. It's often not a problem when the TTL is low. I would let the user do what he wants without warning.